python/triton/runtime/autotuner.py

```diff
@@ -114,7 +114,7 @@ def kernel_call():
                 return bench_res
             return do_bench(kernel_call, warmup=self.num_warmups, rep=self.num_reps, quantiles=(0.5, 0.2, 0.8))
```

Comment:
It returns three results with purpose. When do_bench has `quantiles` not None, it will return timing in [0.5, 0.2, 0.8]th quantile

Reply:
May I know in which case you got an error?

IMO a better fix will have to make `do_bench_cudagraph`, `do_bench`, and `OutOfResources` return consistent results.

In corner case, `do_bench` may return a single result when there's only a single time even when `quantiles` is not None.

So maybe we could remove `quantiles` here.

Reply:
We probably also want to explicitly test out of resources cases in `test_autotuner.py`

Reply:
Ok, missed it, thanks.
https://github.com/openai/triton/pull/3691/files#diff-38f97f1adad5d89076b35408bdda57df7c22f3ffb8a028ec151b70fc3f36978dR141
this line failed.
I'll fix it differently.

Reply:
Thanks! Could you please help me review this related PR? :)  
https://github.com/openai/triton/pull/3690